pkg/define/image/v1/sealer_image_extension.go

```diff
@@ -49,6 +51,23 @@ type ImageSpec struct {
 	OCIv1 ociv1.Image `json:"ociv1"`
 
 	ImageExtension
+
+	// ContainerImageList the container image list contained in the sealer image
+	//
+	// TODO: populate this value during the build phase
+	ContainerImageList []*ContainerImage `json:"containerImageList,omitempty"`
```

Comment:
how about:   map[AppName][]string{images}

Reply:
I don't think so. Here's a case, a container image may not belong to any app.